README.md

```diff
@@ -0,0 +1,177 @@
+    - checking if the 'terc' column includes the correct type (7 digits) of TERC codes
+    - checking if the 'rodzaj_zam_budowlanego' column includes all 4 expected types of building construction intention.
+3. **unzipped_data_uploader_task** - this task uploades the validated data (a table named "reporting_results2020") to a BigQuery database.
+4. **aggregates_creation_task** - this task uses the validated data to create a customized aggregates. The aggregates table consists of columns including information about how many building actions were reported to the polish Main Building Supervision Office in the last 3, 2 and 1 month (divided also by building categories types). At the end, the task sends the aggregates table to the BigQuery database.
+5. **send_email_task** - this task sends a reporting email to a certain mail address. It has the validation report from "validation task" (in .html format).
+
+This DAG is scheduled to be executed once per month, on a first day of each month. It does catchups if necessary.
+
+Moreover, for this project, I created visualisations basing on the "reporting_results2020" table in Jupyter Notebook. The visualisations include various plots and GIF file showing the changes in the number of building permissions per last 3, 2 and 1 month (the GIF is below)
+
+![IMG VOIVOD_CHANGES321](https://github.com/JakBiel/README_img/blob/main/voivodeships.gif)
+
+Also, as a part of the project, I created related visualisations in Looker Studio.
+
+## Docker containers runned in this project
+
+- **airflow-webserver-1** - to present a web page that is a GUI to use Apache Airflow
+- **airflow-scheduler-1** - a scheduler of scheduled DAGs in Apache Airflow
+- **airflow-init-1** - to initialize Apache Airflow server
+- **flower-1** - a support container by initialization of Apache Airflow server
+- **jupyter-1** - to code and present visualisations done in Jupyter Notebook
+- **extending-ariflow-builder-1** - the parent container responsile for starting all the other Apache Airflow containers
+- **postgres-1** - a container to keep Apache Airflow server's data (e.g. xcom)
+
+
+## How to run locally?
+
+To run this project, a few activities will be necessary:
+
+- Clone the project
+- Go to the project directory. Type in the terminal (all commands from this chapter will be for Windows OS):
+
+```bash
+gci -Name
+```
+
+- You should see this:
+
+```bash
+airflow_dockerfile
+dags
+zip_data_jupyter_notebook
+.gitignore
+docker-compose.yaml
+README.md
+```
+
+- Now, if you want to run the project using a new dedicated virtual environment, please type the command below in your terminal (I presume, that you have already installed Python in your global virtual environment). If you want to use your global virtual environment to run this project, please skip these 2 steps below.
+
+```bash
+python -m venv my_new_virtual_environment
+```
+and:
+
+```bash
+.\my_new_virtual_environment\Scripts\activate
```

Comment:
venv creation and activation is not needed

Reply:
ok, I will remove it at all